src/main/res/values/strings.xml

```diff
@@ -1267,13 +1267,13 @@
 
 	<!-- 2020-08-18 -->
 	<string name="pref_behaviour_video_zoom_default_key" translatable="false">pref_behaviour_video_zoom_default</string>
-	<string name="pref_behaviour_video_zoom_default_title">Zoom into videos by default</string>
+	<string name="pref_behaviour_video_zoom_default_title">Zoom in on videos by default</string>
```

Comment:
Preferred original

Reply:
You don't zoom into something. Certainly not something flat.

Reply:
After giving this some thought we should entirely change this to "Expand videos to fill screen" :smile: